sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -716,9 +556,97 @@ const alwaysBeExistingRule = "alwaysbeexistingrule";
       subscriptionName: "alwaysBeExistingSubscription2",
       topicName: "alwaysBeExistingTopic1"
     }
+  },
+  {
+    subscriptionName: "alwaysBeExistingSubscription20",
+    topicName: "alwaysBeExistingTopic1",
+    testCaseTitle: "pass in entity name for forwardTo and forwardDeadLetteredMessagesTo",
+    input: {
+      lockDuration: "PT5M",
+      maxDeliveryCount: 20,
+      forwardDeadLetteredMessagesTo: "alwaysBeExistingTopic2",
+      forwardTo: "alwaysBeExistingTopic2",
+      defaultMessageTtl: "P2D",
+      deadLetteringOnFilterEvaluationExceptions: false,
+      deadLetteringOnMessageExpiration: true,
+      enableBatchedOperations: false,
+      requiresSession: false
+    },
+    output: {
+      lockDuration: "PT5M",
+      maxDeliveryCount: 20,
+      defaultMessageTtl: "P2D",
+      deadLetteringOnFilterEvaluationExceptions: false,
+      deadLetteringOnMessageExpiration: true,
+      enableBatchedOperations: false,
+      requiresSession: false,
+
+      forwardDeadLetteredMessagesTo: `${endpointWithProtocol}alwaysBeExistingTopic2`,
+      forwardTo: `${endpointWithProtocol}alwaysBeExistingTopic2`,
+      autoDeleteOnIdle: "P10675199DT2H48M5.4775807S",
+
+      defaultRuleDescription: undefined,
+
+      messageCount: 0,
+      enablePartitioning: undefined,
+      maxSizeInMegabytes: undefined,
+      sizeInBytes: undefined,
+
+      userMetadata: undefined,
+      messageCountDetails: undefined,
+      entityAvailabilityStatus: "Available",
+      status: "Active",
+
+      subscriptionName: "alwaysBeExistingSubscription20",
+      topicName: "alwaysBeExistingTopic1"
+    }
+  },
+  {
+    subscriptionName: "alwaysBeExistingSubscription21",
+    topicName: "alwaysBeExistingTopic1",
+    testCaseTitle: "pass in absolute URI for forwardTo and forwardDeadLetteredMessagesTo",
+    input: {
+      lockDuration: "PT5M",
+      maxDeliveryCount: 20,
+      forwardDeadLetteredMessagesTo: `${endpointWithProtocol}alwaysBeExistingTopic2`,
+      forwardTo: `${endpointWithProtocol}alwaysBeExistingTopic2`,
+      defaultMessageTtl: "P2D",
+      deadLetteringOnFilterEvaluationExceptions: false,
+      deadLetteringOnMessageExpiration: true,
+      enableBatchedOperations: false,
+      requiresSession: false
```

Comment:
Since we added this test case for testing the case of absolute urls being passed to the forward feature related properties, do we have to set all the other input properties?

Can we set only `forward` and `forwardDeadLetteredMessagesTo` in the `input` here? I see 2 advantages with this
- The test case title and input would be easier to related to each other at a quick glance
- We get coverage for the case when only few of the input properties are set.

Reply:
Makes sense, updated!